composer.json

```diff
@@ -20,7 +20,6 @@
         "symfony/framework-bundle": "~2.5|~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8",
```

Comment:
Looks like the PR on prophecy's side has been merged.

Can you add phpunit back as dev dependency before I merge this?